.drone.yml

```diff
@@ -3,40 +3,146 @@ workspace:
   path: src/github.com/drone-plugins/drone-git
 
 pipeline:
+  deps:
+    image: golang:1.10
+    pull: true
+    commands:
+      - go get -u github.com/golang/dep/cmd/dep
+      - dep ensure
+
   test:
-    image: golang:1.9
+    image: golang:1.10
     pull: true
     commands:
-      - go vet
-      - go test -cover -coverprofile=coverage.out
+      - go vet ./...
+      - go test -cover ./...
 
   build_linux_amd64:
-    image: golang:1.9
+    image: golang:1.10
     pull: true
     group: build
     environment:
       - GOOS=linux
       - GOARCH=amd64
       - CGO_ENABLED=0
     commands:
-      - go build -v -ldflags "-X main.build=${DRONE_BUILD_NUMBER}" -a -o release/linux/amd64/drone-git
+      - |
+        if test "${DRONE_TAG}" = ""; then
+          go build -v -ldflags "-X main.build=${DRONE_BUILD_NUMBER}" -a -o release/linux/amd64/drone-git
+        else
+          go build -v -ldflags "-X main.version=${DRONE_TAG##v} -X main.build=${DRONE_BUILD_NUMBER}" -a -o release/linux/amd64/drone-git
+        fi
+
+  # build_linux_i386:
```

Comment:
Commented out?

Reply:
This also requires a native agent because system packages are involved. We can cross build only go binary images.